packages/mask/src/plugins/Predict/Azuro/PlaceBetDialog.tsx

```diff
@@ -0,0 +1,235 @@
+                                </Grid>
+                                <Grid className={classes.infoContainer} container justifyContent="space-between">
+                                    <Typography className={classes.infoTitle}>Outcome:</Typography>
+                                    <Typography>{outcome}</Typography>
+                                </Grid>
+                                <Grid className={classes.infoContainer} container justifyContent="space-between">
+                                    <Typography className={classes.infoTitle}>Odds:</Typography>
+                                    <Typography>
+                                        {!amount ? (
+                                            condition.value
+                                        ) : actualRateLoading ? (
+                                            <LoadingAnimation />
+                                        ) : (
+                                            actualRate
+                                        )}
+                                    </Typography>
+                                </Grid>
+                                <Grid className={classes.infoContainer} container justifyContent="space-between">
+                                    <Typography className={classes.infoTitle}>Min odds:</Typography>
+                                    <Typography>
+                                        {!amount ? (
+                                            condition.value.toFixed(2)
+                                        ) : minRateLoading || actualRateLoading ? (
+                                            <LoadingAnimation />
+                                        ) : (
+                                            minRate
+                                        )}
+                                    </Typography>
+                                </Grid>
+                                <Grid
+                                    container
+                                    justifyContent="space-between"
+                                    alignItems="center"
+                                    className={classes.infoContainer}>
+                                    <Typography className={classes.infoTitle}>Slippage:</Typography>
+                                    <Typography>
+                                        <ToggleButtonGroup value={slippage} exclusive onChange={handleSlippage}>
+                                            <ToggleButton value={2}>2%</ToggleButton>
+                                            <ToggleButton value={3}>3%</ToggleButton>
+                                            <ToggleButton value={10}>10%</ToggleButton>
+                                        </ToggleButtonGroup>
+                                    </Typography>
+                                </Grid>
+                            </Grid>
+                            <WalletConnectedBoundary>
```

Comment:
Do you need `<ChainBoundary />`?

Reply:
Indeed! better with `<ChaindBoundary />`.

Reply:
Removed use of `ChaindBoundary`. Not necessary.

Reply:
Could you please tell us the reason? The user may choose a non-xDai network.